inc/zoo/swar/SWAR.h

```diff
@@ -430,13 +430,49 @@ booleans(SWAR<NB, T> arg) noexcept {
 template<int NBits, typename T>
 constexpr auto
 differents(SWAR<NBits, T> a1, SWAR<NBits, T> a2) {
-    return booleans(a1 ^ a2);
+    return ~equals(a1, a2);
 }
 
+/*
+ * Returns a BooleanSWAR that contains a true value in each lane where the two
+ * inputs SWARs match.
+ */ 
```

Comment:
Nice.
If it is the same to you, please use Doxygen/Autodoc format of `/// \return or /** \return or /// @return` etc.
There are several tools, including XCode that automatically parse those comments

Reply:
There is exactly one other place, in the new benchmarks, this is done.  Is this a new request, that the library start supporting Autodoc/Doxygen formats?  If we'd like to move the code over to that, sounds like something for a standalone PR.

Reply:
We should do it for all new code, in a measured, reasonable way.
Like I said, if it is the same to you, you may as well use doxygen in particular.
Is there agreement?

Reply:
I don't know why you say "[t]here is exactly one other place [...] this is done": this codebase, as a whole, as been using Doxygen style comments from the very beginning, even in the original Pokerbotic where the SWAR originated.

Reply:
```enkiwa@penguin ~/src/zoo$ find . | xargs grep -i -s '@return'
xargs: unmatched single quote; by default quotes are special to xargs unless you use the -0 option
./benchmark/atoi_impl.h:/// @return a pair to indicate the aligned pointer to the base of the block
enkiwa@penguin ~/src/zoo$ find . | xargs grep -i -s '\\\\return'
xargs: unmatched single quote; by default quotes are special to xargs unless you use the -0 option```
I see exactly one place we've specified a return value cleanly, in a Doxygen understandable way. That was the point of my comment.

Sure, I can start using doxygen readable return blocks.  The point of the previous statement was "they were so absent previously I didn't even realize it was something we cared about".

Reply:
Come on, Scott:
on `inc`,
` find . -name '*.h' -exec grep "^[[:space:]]*//[/*]" {} +`
Gives over 200 matches, and this undercounts because all of the post-documenting comments `///<` are not included.  There are many Doxygen commands.
And my original point is that if it is the same to you, please use Doxygen.